themes/admin/Sea_Green/css/sea_green-rtl.css

```
@@ -16497,6 +16497,33 @@ body {
+a:focus,
+button:focus,
+input:focus,
+select:focus,
+textarea:focus,
+summary:focus,
+details:focus,
+[tabindex]:not([tabindex="-1"]):focus,
+[contenteditable]:not([contenteditable=false]):focus,
+iframe:focus,
+area:focus {
+  outline: 2px solid currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)